server-worker/src/main/java/com/aliyun/emr/rss/service/deploy/worker/storage/PartitionFilesSorter.java

```diff
@@ -413,53 +465,75 @@ static class ShuffleBlockInfo {
   }
 
   class FileSorter {
```

Comment:
How about we create a FileSorter trait, then implement a HdfsFileSorter, a LocalFileSorter etc, them the code won't be a mess.